### DISCUSSION TITLE
[FEEDBACK] How to donate money via Github?

### DISCUSSION THREAD

**Discussion by Robert-Ernst** [Lissy93/dashy#90] [Q&A] [answered]
I like to donate a small token of appreciation of those Repositories I'm using.  
The Github payment function is for me quite neat for that.  
Do you want to enable that functionality here?
https://github.com/sponsors

**Comment by lissy93** [comment, marked as answer]
Hi @Robert-Ernst - that's super kind of you, thank you so much! 
I've created a sponsor profile: https://github.com/sponsors/Lissy93

**Comment by lissy93** [comment]
Thank you so much @Robert-Ernst - Very kind of you ❤️
